modules/tensor_mechanics/src/materials/RecomputeRadialReturnIsotropicPlasticity.C

```diff
@@ -6,10 +6,12 @@
 /****************************************************************/
 #include "RecomputeRadialReturnIsotropicPlasticity.h"
 
+#include "Function.h"
```

Comment:
Did this include need to be added?  I didn't see any other changes in the file that would appear to require it.